var/spack/repos/builtin/packages/bazel/gcc11_2.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/third_party/ijar/zlib_client.h b/third_party/ijar/zlib_client.h
```

Comment:
Any reason not to combine 1 and 2 into a single patch?

Reply:
No, the reason they are separate was just to be consistent with the source the patches were derived from. That is also why they are named the way they are. I thought that would be the most clear way to track what is going on.